src/lambdas/initiateAthenaQuery/handler.test.ts

```diff
@@ -1,21 +1,6 @@
 import { handler } from './handler'
-import * as initiateQueryImportWrapper from './handler'
-import { confirmAthenaTable } from './confirmAthenaTable'
-import { startQueryExecution } from './startQueryExecution'
-import {
-  testAthenaQueryEvent,
-  testManualAthenaQueryEvent
-} from '../../utils/tests/events/initiateAthenaQueryEvent'
-import { updateZendeskTicketById } from '../../sharedServices/zendesk/updateZendeskTicket'
-import { getDatabaseEntryByZendeskId } from '../../sharedServices/dynamoDB/dynamoDBGet'
-import { createQuerySql } from './createQuerySql'
-import { updateQueryByZendeskId } from '../../sharedServices/dynamoDB/dynamoDBUpdate'
-import { ConfirmAthenaTableResult } from '../../types/athena/confirmAthenaTableResult'
-import {
-  dataPathsTestDataRequest,
-  noIdTestDataRequest,
-  testDataRequest
-} from '../../utils/tests/testDataRequest'
+import * as initiateQueryImportWrapper from './initiateQuery'
```

Comment:
You don't need this any more, or the spy

Reply:
done